src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -147,9 +156,27 @@ export default {
 					variables: prefetchedRecommendedLoansVariables
 				});
 
+				const matchedLoansPromise = client.query({
```

Comment:
Why are we prefetching the second row? Is it ever visible above the fold when the page first loads?

Reply:
I think it depends of screen resolution, its ever visible for the test for myself

<img width="1294" alt="Screenshot 2023-05-04 at 14 19 47" src="https://user-images.githubusercontent.com/94026278/236320202-331d1a48-fc39-417a-88f8-6f477d66aef2.png">

Reply:
How much do the extra prefetch queries affect the Lighthouse scores locally?

Reply:
Nice call! Ill test it in a while...